api/configuration/menuConfigs/configs.ts

```diff
@@ -0,0 +1,57 @@
+import { baseConfig } from './base-config';
+import { aatDifferences } from './aat-diffs';
+import { previewDifferences } from './preview-diffs';
+
+export function setupMenuConfig(environment) {
+  let config;
+  if (environment === 'prod') {
+    config = baseConfig;
+  } else if (environment === 'aat') {
+    let aatConfig = { ...baseConfig };
+    const rolesRegexChanges = {
```

Comment:
This bit is just a screw up really. It should be (judge)|(judiciary)|(panelmember) in all configs
    }

Reply:
You can leave it for now if you like, or if you want to you can just eliminate that difference